spec/ace.md

```diff
@@ -130,8 +130,69 @@ An executor MAY implement a "strict mode" where an image cannot run unless all i
 
 ### Linux Isolators
 
-These isolators are specific to the Linux kernel and are impossible to represent as a 1-to-1 mapping on other kernels.
-The first example is "capabilities" but this will be expanded to include things such as SELinux, SMACK or AppArmor.
+These isolators are specific to the Linux kernel and are impossible to re wpresent as a 1-to-1 mapping on other kernels.
+This set includes Linux-specific isolators, which may relies on kernel technologies like seccomp, SELinux, SMACK or AppArmor.
+
+#### os/linux/seccomp-remove-set
+
+* Scope: app
+
+**Parameters:**
+
+* **set** case-sensitive list of syscall names that will be blacklisted (ie. blocked); values starting with `@` MUST be handled as scoped special values (see notes below). All syscalls specified in this set MUST be blocked. This set MAY be augmented by an implementation-specific default blacklist set (see notes below). Syscalls not specified in the union of these two sets MUST NOT be blocked.
+* **errno** all-uppercase name of a single [errno code](http://man7.org/linux/man-pages/man3/errno.3.html) that will be returned by blocked system calls, instead of terminating. If missing or empty, by default blocked syscalls MUST result in app termination via `SIGSYS` signal.
+
+**Notes:**
+ 1. The `os/linux/seccomp-remove-set` isolator cannot be used in conjunction with the `os/linux/seccomp-retain-set` isolator.
```

Comment:
The spec should say whether having several isolators of type `os/linux/seccomp-remove-set` is allowed. I guess we should not allow that: otherwise, conflicting errno for the same syscalls would make it ambiguous.

Ditto for the retain-set.